polymer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "entrypoint": "index.html",
```

Comment:
Not needed - honestly I thought it was needed to run "polymer serve" but it is not :-)